model/metrics/vcs.yaml

```diff
@@ -0,0 +1,150 @@
+groups:
+  - id: metric.vcs.repository.change.count
+    type: metric
+    metric_name: vcs.repository.change.count
+    brief: 'The number of changes (pull requests) in a repository, categorized by their state (either open or merged)'
+    instrument: gauge
+    unit: "{change}"
+    stability: experimental
+    #extends: metric_attributes.vcs
+    attributes:
+      - ref: vcs.change.state
+        requirement_level: required
```

Comment:
For now all VCS metric attributes were defined as required. Is that fine or should we change it to recommended?

Reply:
Discussed in CICD SIG meeting 2024-10-03:

For now we leave all metric attributes as required, because usually all attributes are required for the metric to be usable.
Later we can discuss on a metric-by-metric basis if every attribute is really required. At that time we might relax the requirement level.

The metric requirement level this is for now tracked only in the markdown file.
In 84b2b6a7aabf447e6a24c8b551cc22450f90efd8 I added recommended for all metrics except opt-in for the `vcs.repository.contributor.count` metric which the githubreceiver had as optional.